### DISCUSSION TITLE
"RENAME" showing up in top right corner when searching in the app list

### DISCUSSION THREAD

**Discussion by xfzv** [DroidWorksStudio/DoNotEnter#26] [Q&A] [answered]
Just wondering what is this:

![1](https://github.com/DroidWorksStudio/mLauncher/assets/78810647/6fd52a45-6ff5-48a0-bb89-e1af10d240ea)

It shows up when starting to type any character in the app list, even if there is no match (as you can see in the screenshot). Long pressing doesn't do anything, a single tap brings me back to the home screen.

It started to show up after updating to 1.4.5 I believe, and is still visible in 1.4.6.

**Comment by CreativeCodeCat** [comment]
Oh that's only meant to happen if you long press an app on the home screen so you can rename them apps I'll have a look into why it's doing it also in app list in the next few days.

**Comment by CreativeCodeCat** [comment, marked as answer]
@xfzv i have fixed it in [commit](https://github.com/DroidWorksStudio/mLauncher/commit/5e8add8240496d2ddd913f078efc2474ec1eea35) will hope to push a full release soon.

I also move it to this repo as I don't want it clogging up the main repo when this should of been under issues

**Reply by xfzv** [reply]
Thanks for the heads-up!

> I also move it to this repo as I don't want it clogging up the main repo when this should of been under issues

I wasn't too sure whether this was an actual issue or just me using wrong settings so I ended up creating a discussion. Sorry about that.
